build/import/Packages.targets

```diff
@@ -89,8 +89,7 @@
     <PackageReference Update="Roslyn.Diagnostics.Analyzers"                                           Version="3.3.2" />
 
     <!-- NuGet -->
-    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="5.8.0-preview.2.6776" />
-    <PackageReference Update="NuGet.VisualStudio"                                                     Version="6.0.0-preview.0.2" />
+    <PackageReference Update="NuGet.SolutionRestoreManager.Interop"                                   Version="6.0.0-preview.2.129" />
```

Comment:
This is just an update. You'll need to find the includes elsewhere and change them too.

Reply:
GitHub's not letting me start a new conversation on the same line of code :(

NuGet merged `NuGet.SolutionRestoreManager.Interop` into `NuGet.VisualStudio`, not the other way around. You need to keep `NuGet.VisualStudio`.

Reply:
Thank you @zivkan